python_configure.bzl

```diff
@@ -2,15 +2,57 @@
 
 `python_configure` depends on the following environment variables:
 
-  * `PYTHON_BIN_PATH`: location of python binary.
+  * `PYTHON_BIN_PATH`: Location of python binary.
   * `PYTHON_LIB_PATH`: Location of python libraries.
+
+These can be directly set with the `python_interpreter_target` and
+`python_library_target` respectively.
 """
 
 _BAZEL_SH = "BAZEL_SH"
 _PYTHON_BIN_PATH = "PYTHON_BIN_PATH"
 _PYTHON_CONFIG_BIN_PATH = "PYTHON_CONFIG_BIN_PATH"
 _PYTHON_LIB_PATH = "PYTHON_LIB_PATH"
 
+# TODO: Move scripts to respective files and expose to rule.
+_INCLUDE_SCRIPT = """
+from __future__ import print_function
+from distutils import sysconfig
+import shutil
+
+python_inc_dir = sysconfig.get_python_inc()
+config_h_path = sysconfig.get_config_h_filename()
+shutil.copyfile(config_h_path, python_inc_dir + "/pyconfig.h")
```

Comment:
Can we somehow avoid this side effect? Wouldn't it fail when using a systemwide installation?

Reply:
Hmm. Yeah I don't remember why I made this change- I imagine I did it for some reason. Without the correct permissions this would crash.
We could check to see whether this is needed or possible- but I think it would be better to just remove this.

Not sure. I can test this at some point, but likely not until next week.

Reply:
Was it from the patch here? https://github.com/pybind/pybind11_bazel/pull/29#issuecomment-785489233
I think it might be better to just have the genrule copy it.

There's no hurry anyway :)